### ISSUE TITLE
db insert issue

### ISSUE THREAD

**Issue by lu-chi** [lanmaster53/recon-ng#85]
Hello there,

I've noticed recenlty on the latest Kali with latest recon-ng 5.1.1 onboarded whenever I do 
db insert domain blahblahblah.com (or any other table) 
it throws the error message as below:
[!] Columns and values length mismatch.

Is there any reason and solution for that to happen?

**Comment by leosaldi** [comment]
First, use "db insert domains" (enter) 
then write the domain

Slds.

**Comment by lu-chi** [comment]
Yep, I've realized it works that way. Is there a reason it changed?
